CHANGELOG.md

```diff
@@ -5,6 +5,16 @@ Note: this log contains only changes from knife-ec2 release 0.8.0 and later
 prior to release 0.8.0, please visit the [source repository](https://github.com/chef/knife-ec2/commits).
 
 
+## Latest Release 0.16.0 (yyyy-mm-dd)
+* [knife-ec2:#503](https://github.com/chef/knife-ec2/pull/503) Update list of instance_types that support ebs-encryption.
+* [knife-ec2:#502](https://github.com/chef/knife-ec2/pull/502) Update bundler due to travis failure.
```

Comment:
This can probably stay out of the changelog

Reply:
Remove `Update bundler due to travis failure.` from changelog. Thanks for review @tas50